ansible_base/oauth2_provider/views/token.py

```diff
@@ -54,4 +55,4 @@ def create_token_response(self, request):
 class OAuth2TokenViewSet(ModelViewSet, AnsibleBaseDjangoAppApiView):
     queryset = OAuth2AccessToken.objects.all()
     serializer_class = OAuth2TokenSerializer
-    permission_classes = [OAuth2TokenPermission]
+    permission_classes = [OAuth2ScopePermission, OAuth2TokenPermission]
```

Comment:
Should this also be wrapped in `try_add_oauth2_scope_permission`?

Reply:
I think no for the same reason as the other one in `ansible_base.oauth2_provider`